src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -130,8 +130,7 @@ public bool AllowIntrospection
         }
     }
 
-    [JsonIgnore]
```

Comment:
Don't you still need this JsonIgnore? what does the output config look like?

Reply:
This field now shows up in output config files which isn't intended: 
![image](https://github.com/Azure/data-api-builder/assets/6414189/ed22b43a-9d67-489e-af62-1c3bc0557874)

Reply:
changed to private so this should be OK.